src/game_functions/board_functions.c

```diff
@@ -2,6 +2,15 @@
 #include <stdbool.h>
 #include <stdlib.h>
 
+#define TILES_AROUND 8
+
+/**
+ * Update all necessary tiles as a certain tile has been clicked.
+ * @param board the Board struct to update.
+ * @param index the index at which the update is to start.
+ */
+void _updateNeighborTiles(Board *board, int index);
```

Comment:
Is this done because it is not to be used outside of the board functions?

Reply:
Yes, that's a way to indicate that in the name of the function. It's a "private" function.